### ISSUE TITLE
`keras`: Include `learning_rate` scalar.

### ISSUE THREAD

**Issue by daavoo** [iterative/dvclive#255]
The [TensorBoard callback](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard) collects and stores learning rate:

https://github.com/keras-team/keras/blob/d8fcb9d4d4dad45080ecfdd575483653028f8eda/keras/callbacks.py#L2527-L2531

**Comment by dberenbaum** [comment]
Closing as duplicate of #317
